src/core/Cargo.toml

```diff
@@ -78,19 +78,23 @@ harness = false
 
 ## Wasm section. Crates only used for WASM, as well as specific configurations
 
-[target.'cfg(all(target_arch = "wasm32", target_vendor="unknown"))'.dependencies.wasm-bindgen]
+[target.'cfg(all(target_arch = "wasm32", target_os="unknown"))'.dependencies.wasm-bindgen]
 version = "0.2.62"
 features = ["serde-serialize"]
 
-[target.'cfg(all(target_arch = "wasm32", target_vendor="unknown"))'.dependencies.web-sys]
+[target.'cfg(all(target_arch = "wasm32", target_os="unknown"))'.dependencies.web-sys]
 version = "0.3.58"
 features = ["console", "File"]
 
-[target.'cfg(all(target_arch = "wasm32", target_vendor="unknown"))'.dev-dependencies]
+[target.'cfg(all(target_arch = "wasm32"))'.dependencies.chrono]
+version = "0.4.23"
+features = ["wasmbind"]
```

Comment:
we don't have a direct dependency on `chrono`, but setting this feature here solves problems in upstream deps